usage/app/lib/Config.scala

```diff
@@ -85,7 +85,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
       .withRegion(Regions.EU_WEST_1)
 
   val postfix = if (stage == "DEV") {
-    iamClient.getUser().getUser().getUserName()
+    "ropnert"
```

Comment:
again needed to put something temporary in so that it would work. discussion is in seperate PR.